packages/arb-token-bridge-ui/src/components/TransactionHistory/TransactionsTableDetailsSteps.tsx

```diff
@@ -0,0 +1,180 @@
+import { ReactNode } from 'react'
+import dayjs from 'dayjs'
+import { twMerge } from 'tailwind-merge'
+import {
+  ArrowTopRightOnSquareIcon,
+  CheckCircleIcon
+} from '@heroicons/react/24/outline'
+
+import { MergedTransaction } from '../../state/app/state'
+import { getExplorerUrl, isNetwork } from '../../util/networks'
+import { useChainLayers } from '../../hooks/useChainLayers'
+import { isTxClaimable, isTxCompleted, isTxPending } from './helpers'
+import { TransactionsTableRowAction } from './TransactionsTableRowAction'
+import { ExternalLink } from '../common/ExternalLink'
+import {
+  WithdrawalCountdown,
+  getTxConfirmationDate
+} from '../common/WithdrawalCountdown'
+import { DepositCountdown } from '../common/DepositCountdown'
+import { useRemainingTime } from '../../state/cctpState'
+
+function getTransferDurationText(tx: MergedTransaction) {
+  const { isTestnet, isOrbitChain } = isNetwork(tx.parentChainId)
+
+  if (tx.isCctp) {
+    return isTestnet ? '1 minute' : '10 minutes'
+  }
+
+  if (!tx.isWithdrawal) {
+    if (isOrbitChain) {
+      return '1 minute'
+    }
+    return isTestnet ? '10 minutes' : '15 minutes'
+  }
+
+  // withdrawals
+  return getTxConfirmationDate({
+    createdAt: dayjs(),
+    withdrawalFromChainId: tx.childChainId
+    // we set from to current time so that we get the full withdrawal confirmation time
+  }).from(dayjs(), true)
+}
+
+function needsToClaimTransfer(tx: MergedTransaction) {
+  return tx.isCctp || tx.isWithdrawal
+}
+
+const Step = ({
+  done = false,
+  claimable = false,
+  pending = false,
+  text,
+  endItem = null
+}: {
+  done?: boolean
+  claimable?: boolean
+  pending?: boolean
+  text: string
+  endItem?: ReactNode
+}) => {
+  // defaults to a step that hasn't been started yet
+  let borderColorClassName = 'border-white/50'
+  let textColorClassName = 'text-white/50'
+
+  if (done || claimable) {
+    borderColorClassName = 'border-green-400'
+    textColorClassName = 'text-green-400'
```

Comment:
Design nit, feel free to ignore or raise with Val - I don't think we should have ALL the done text as green. The text can be white and just the checkmark-icon can be green in happy cases.

Otherwise it's looking way too colorful and grabbing all my attention IMHO.

Reply:
Raised, will update